scripts/e2e-sinopia.config.yml

```diff
@@ -19,3 +19,5 @@ packages:
 
 logs:
   - {type: file, path: sinopia.log, format: pretty, level: http}
+
+max_body_size: '50mb'
```

Comment:
Nice! The package grew a bit because of the iOS template, right? I'm surprised it grew significantly enough to cause an upload issue.